src/components/atoms/SpacesDropdown/SpacesDropdown.tsx

```diff
@@ -41,6 +44,11 @@ export const SpacesDropdown: React.FC<SpacesDropdownProps> = ({
   // doesn't exist on SPACE_PORTALS_ICONS_MAPPING
   const filteredPortals = omit(portals, VenueTemplate.playa);
 
+  const portalOptions = useMemo(
+    () => ({ ...filteredPortals, ...spaceNoneOption }),
```

Comment:
When I've seen this `None` option on other websites, usually it's in the beginning of the dropdown, not at the end. If this is per design, dismiss my comment. Otherwise, I think we should ask product.

Reply:
thanks, put `none` to the top of the list 06cbf3a